sympy/physics/continuum_mechanics/beam.py

```diff
@@ -210,6 +213,16 @@ def reaction_loads(self):
         """ Returns the reaction forces in a dictionary."""
         return self._reaction_loads
 
+    @property
+    def rotation_jumps(self):
+        """ Returns the rotation jumps in rotation hinges in a dictionary."""
```

Comment:
It isn't so clear what this returns. Is a "jump" a symbol? expression? number? Making the docstring a bit more informative would help.

Reply:
Changed the docstring to: 
"""
Returns the value for the rotation jumps in rotation hinges in a dictionary.
The rotation jump is the rotation (in degrees) in a rotation hinge. This can
be seen as a jump in the slope plot.
"""

Same for the deflection jump. Is it okay like this?